examples/all-clusters-app/nrfconnect/configuration/nrf52840dk_nrf52840/pm_static_dfu.yml

```diff
@@ -1,27 +1,27 @@
 mcuboot:
     address: 0x0
-    size: 0x7000
+    size: 0x6800
```

Comment:
I don't think you can do this - sector size on nRF52 is 4k so you won't be able to properly erase the partition with these changes. Could you simply disable testing DFU in CI for all-clusters-app (see the change in https://github.com/project-chip/connectedhomeip/pull/26566/files)?